ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -0,0 +1,154 @@
+
+# CSV Config
+# Create the respective directory as provided in the path below (Path to folder where reports will be stored)
+CSV_REPORTS_PATH={{ml_survey_csv_reports_path | default("reports")}}
+
+# Kafka Configuration
+# Flag which define where communication with kafka is on or off 
+KAFKA_COMMUNICATIONS_ON_OFF={{ml_survey_kafka_communications_on_off | default("ON")}}
+
+# IP address of kafka server with port without HTTP
+KAFKA_URL={{ml_survey_kafka_url}} 
+
+# Kafka topic name for pushing completed sumbmission
+COMPLETED_SUBMISSION_TOPIC={{ml_survey_completed_submission_topic}}
+
+# Kafka topic name for pushing incompleted submission
+INCOMPLETE_SUBMISSION_TOPIC={{ml_survey_incomplete_submission_topic}}
+
+# Kafka topic name for pushing submissions for which rating has to be done
+SUBMISSION_RATING_QUEUE_TOPIC={{ml_survey_submission_rating_queue_topic}}
+
+# Kafka topic name for pushing completed observation submission
+COMPLETED_OBSERVATION_SUBMISSION_TOPIC={{ml_survey_completed_observation_submission_topic}}
+
+# Kafka topic name for pushing incomplete observation submission
+INCOMPLETE_OBSERVATION_SUBMISSION_TOPIC={{ml_survey_incomplete_observation_submission_topic}}
+
+# Kafka topic name for pushing notification messages
+NOTIFICATIONS_TOPIC={{ml_survey_notifications_topic}}	
+
+# Kafka topic name for completed survey submission
+COMPLETED_SURVEY_SUBMISSION_TOPIC={{ml_survey_completed_survey_submission_topic}}
+
+# Kafka topic name for incomplete survey submission
+INCOMPLETE_SURVEY_SUBMISSION_TOPIC={{ml_survey_incomplete_survey_submission_topic}}
+
+# Kafka consumer group for ML Survey Service
+KAFKA_GROUP_ID={{ml_survey_kafka_group_id}}
+
+#  Kafka topic name for pushing project submission related data
+ML_PROJECT_SERVICE_SUBMISSION_TOPIC={{ml_survey_improvement_project_submission_topic | default(env+".ml.survey.project.submission")}}
+
+# Elastic search configurations 
+# Elastic search enable/disable flag E.g. ON/OFF
+ELASTICSEARCH_COMMUNICATIONS_ON_OFF={{ml_survey_elasticsearch_communications_on_off | default("ON")}}
+
+# Elastic search host url
+ELASTICSEARCH_HOST_URL={{ml_survey_elasticsearch_host_url}}
+
+# Elastic search request timeout in miliseconds
+ELASTIC_SEARCH_REQUEST_TIMEOUT={{ml_survey_elastic_search_request_timeout}}
+
+# Elastic search maximum number of retries before dropping connection
+ELASTIC_SEARCH_MAX_RETRIES={{ml_survey_elastic_search_max_retries}}
+
+# Flag to obtain ES Nodes list at the time of connection e.g. true/false
+ELASTIC_SEARCH_SNIFF_ON_START={{ml_survey_elastic_search_sniff_on_start}} 
+
+# Elastic search index name for storing user data
+ELASTICSEARCH_USER_EXTENSION_INDEX={{ml_survey_elasticsearch_user_extension_index}}
+
+# Elastic search index name for storing entity data
+ELASTICSEARCH_ENTITIES_INDEX={{ml_survey_elasticsearch_user_extension_index_type}}
+
+# ML Core Service URL
+ML_CORE_SERVICE_URL={{ml_survey_improvement_project_base_url}}
```

Comment:
what's the default value for this ?

Reply:
changed like this ML_CORE_SERVICE_URL={{ml_core_service_url} 
value developer ask us to check from our end for sunbird though it is internal service